src/conf_mode/interfaces-nat64.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2019-2021 VyOS maintainers and contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 or later as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import shutil
+from sys import exit
+
+from vyos.config import Config
+from vyos.configdict import get_interface_dict
+from vyos.configverify import verify_vrf
+from vyos.configverify import verify_address
+from vyos.ifconfig import NAT64If
+from vyos import ConfigError
+from vyos import airbag
+from vyos.template import render
+from vyos.util import call
+airbag.enable()
+
+
+def get_config(config=None):
+    """
+    Retrive CLI config as dictionary. Dictionary can never be empty, as at least the
+    interface name will be added or a deleted flag
+    """
+    if config:
+        conf = config
+    else:
+        conf = Config()
+    base = ['interfaces', 'nat64']
+    nat64 = get_interface_dict(conf, base)
+    return nat64
+
+def verify(nat64):
+    if 'deleted' in nat64:
+        # bail out early
+        return None
+
+    verify_vrf(nat64)
+
+    return None
+
+def generate(nat64):
+    if 'deleted' in nat64 or 'disable' in nat64:
+        return None
+    ifname = nat64['ifname']
+    config_file = f'/run/tayga/{ifname}.conf'
+    render(config_file, 'tayga/tayga.conf.j2', nat64)
+    return None
+
+def apply(nat64):
+    ifname = nat64['ifname']
+    d = NAT64If(ifname)
+    if 'deleted' in nat64 or 'disable' in nat64:
+        call(f'systemctl stop tayga@{ifname}.service')
+        if 'deleted' in nat64:
+            d.remove()
+            config_file = f'/run/tayga/{ifname}.conf'
+            os.unlink(config_file)
```

Comment:
Please check if the configuration really exists before unlinking to avoid any exception

Reply:
done